tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -68,6 +68,7 @@ internal enum IntegrationId
         AwsDynamoDb,
         HardcodedSecret,
         IbmMq,
+        Remoting,
```

Comment:
Don't forget the dd-go PR 😄

Reply:
Done: https://github.com/DataDog/dd-go/pull/110722